### ISSUE TITLE
Database model reconsideration

### ISSUE THREAD

**Issue by shari09** [shift-board/shiftboard#15]
Right now because of how the photo is stored as a blob, it can either:
- converted into a base64 string each time to send the image, which creates the following problems:
  - base64 is much greater than binary data
  - it can be easily cached in public wifis (I read this somewhere)
  - if you were to inspect, it is *very messy* since it's **huge**
- store the file `name` and `created_at` attributes, the post will contain `name`, `message`, `photo.url` and `photo.name`. The url will be used as the `src` attribute in the `<img />`.
It'll pass back a JSON obj like this:
```js
{
  name: string;
  message: string;
  photo: {
    url: string;
    name: string;
  }
}
```

**Comment by EmeraldEntities** [comment]
I'll think about the connections between the API, frontend, and database and see how we can resolve this issue.

The database should be remodelled to move the images into a separate table though, since it apparently helps with querying according to a few StackOverflow posts I've read (which clearly are the most important and reliable sources).

**Comment by EmeraldEntities** [comment]
Going to update the database so that Image BLOBs are no longer stored in the same table as the `Board` model to promote search efficiency.

**Comment by shari09** [comment]
Can you also add a `created_at` timestamp for the posts/boards, just so when the server were to load them it won't be in arbitrary order.

**Comment by EmeraldEntities** [comment]
Yeah, sure thing!

**Comment by EmeraldEntities** [comment]
This should be addressed by #17

**Comment by shari09** [comment]
Great! I'll link the PR and then close this issue
